packages/react-scripts/config/jest/react-intlMock.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const React = require('react');
+const intlShape = require('react-intl').intlShape;
+
+const addLocaleData = () => {};
+const FormattedMessage = () => React.createElement('div');
```

Comment:
Does this mean that if we change the key or params the test will still pass if we do matchSnapshot?

Reply:
No, in the snapshots you still see the component name and props.
This is how it looks in the snapshots:

```jsx
<FormattedMessage
        id="common.alerts"
      />
```